ROMFS/px4fmu_common/init.d/rc.uuv_apps

```diff
@@ -20,6 +20,16 @@ ekf2 start &
 #
 uuv_att_control start
 
+
+#
+# Start UUV Position Controller for UUV with 6DOF
+#
+if [ $UUV_DOF_ACTUATED = 6 ]
+then
+	uuv_pos_control start
```

Comment:
Why is the position control module not started when the it is not fully actuated(DOF 6)? Wouldn't it be better to also be able to control position for underactuated UUVs?

Reply:
uuv_pos_control does not feature support for the 4dof yet.
will be one of the next steps.

Reply:
@DanielDuecker Yes, but if this is the plan, what I meant was that it would be better if we can handle this from inside the module, rather than preventing the whole app from starting.